examples/mt-pytorch/client.py

```diff
@@ -31,6 +31,7 @@ def evaluate(self, parameters, config):
 
 # Start Flower client
 fl.client.start_numpy_client(
-    server_address="[::]:9092",
+    server_address="0.0.0.0:9093",
```

Comment:
Should we mention that when using "use_rest", one needs to use port 9093 by default, or should we always use the same port on the server side so users on the local host only need to change the boolean flag? Wdyt?

Reply:
Both approaches are valid. In the current version, different communication stacks run on different ports (by default) because we might have multiple stacks running together at some point. Happy to change it though.

Reply:
Fine for me either way. Let's keep it and see what comes back from users.